src/demo/index.php

```diff
@@ -26,7 +26,7 @@ function gtag() { dataLayer.push(arguments); }
     <script async defer src="https://buttons.github.io/buttons.js"></script>
 </head>
 
-<body <?php echo $_COOKIE["darkmode"] == "on" ? 'data-theme="dark"' : ""; ?>>
+<?php echo (isset($_COOKIE["darkmode"]) && !empty($_COOKIE) && $_COOKIE["darkmode"] == "on") ? 'data-theme="dark"' : ""; ?>
```

Comment:
body tag was removed

```suggestion
<body <?php echo (isset($_COOKIE["darkmode"]) && $_COOKIE["darkmode"] == "on") ? 'data-theme="dark"' : ""; ?>>
```

Reply:
Sorry, I copied the suggested change from the prior review change request comment. It seems to be correct now

Reply:
Oh, I guess that was my mistake earlier, then